content/docs/details/permissions/index.md

```diff
@@ -6,46 +6,31 @@ description: How to configure access control permissions in Roadie.
 
-## Assigning Roles
+A good example of a permission is the ability to allow users to see a Scaffolder Template. Let's say this Template has the ability to change a cloud budget for a given service. Not everyone within an organisation may have that ability. A permissions attached to that Template would allow the team governing Roadie to limit access to that Template. 
 
-There are two ways to assign a role to a user in Roadie:
+## Features
 
-- Assign roles in the user management screen
-- Provide a `roles` field in your id token provided to Roadie during login.
+- **User Management** user roles can be managed via the no-code Roadie Admin UI or ingested from an Identity Provider via the identity token.
+- **Custom Roles** can be created to manage subsets of users
+- **Custom Policies** can be created to cater for specific permissions
+- **Custom Permissions** can be created and consumed in Custom Policies (from a custom plugin for example)
 
-### User Management
+## Structure
 
-- Visit the user management section in `https://<tenant name>.roadie.so/administration/settings/manage-users`.
-- Find the name of the user you would like to assign a role to.
-- Click the edit pencil and then add the roles you would like to assign the user.
+Roadie permissions are made up:
 
-### Roles from Identity providers id token
+- **Users**: users have Roles which grant them permissions to execute certain tasks within Roadie or view certain information
+- **Roles**: which are attached to users. They are groups of policies. 
+- **Policies**: which are groups of permissions, rolled up into Roles. 
+- **Permissions**: granular code-level gates that evaluate whether an individual users has access to a given piece of functionality.
 
-If your Roadie tenant is using a custom identity provider, you can choose to include the roles field in your id token as follows:
+## Backstage Permissions Framework
 
-```json
-{
-  "sub": "77d0c4cb-706c-4aa4-b18e-bed538a33aa7",
-  "roles": ["viewer", "tech-insights-admin"]
-}
-```
+Roadies Permissions system is based on the Bacsktage Permissions framework. More information on the framework in general can be found in Backstage docs:
 
-## Setting the default roles
-
-If a user has no role associated yet, e.g. if it is the first time they have logged in and there are no roles associated with the user, then the user is allowed the policies associated with the default roles. There can be many default roles.
-
-To edit the default roles:
-
-- Visit `http://<tenant name>.roadie.so/administration/settings/roles`
-- Click the edit pencil beside the roles you would like to be the default roles
-- The click "Set as default role" and save.
-- You can perform similar steps to make a role no longer a default role.
+- Backstage docs on [policy definitions](https://backstage.io/docs/permissions/writing-a-policy/).
```

Comment:
I believe these links are irrelevant, we have implemented our own permissions policy.. I would suggest removing these links. I think these are docs that integrators would use.. and we wouldnt expect our users to have this context. IMO thoughts?

Reply:
What I want to indicate is (mainly for self-hosted folks with custom plugins) that the permissions framework is there and our work is based on it. We deviate + you don't need to understand permissions in order to understand our stuff, but it's useful context

Reply:
coola boola